src/table.ts

```diff
@@ -2237,6 +2246,129 @@ class Table extends common.ServiceObject {
     const body = Table.formatMetadata_(metadata as TableMetadata);
     super.setMetadata(body, callback!);
   }
+
+  getIamPolicy(
+    optionsOrCallback?: GetPolicyOptions | PolicyCallback
+  ): Promise<PolicyResponse>;
+  getIamPolicy(options: GetPolicyOptions, callback: PolicyCallback): void;
+  /**
+   * Run a query scoped to your dataset.
+   * @returns {Promise}
+   */
+  getIamPolicy(
+    optionsOrCallback?: GetPolicyOptions,
+    cb?: PolicyCallback
+  ): void | Promise<PolicyResponse> {
+    const options =
+      typeof optionsOrCallback === 'object' ? optionsOrCallback : {};
+    const callback =
+      typeof optionsOrCallback === 'function' ? optionsOrCallback : cb;
+
+    if (
+      options.requestedPolicyVersion &&
```

Comment:
Just to double check - `options.requestedPolicyVersion` will return falsey values for `null`, `undefined`, `0`, and empty strings.  That may be fine here, but I like to make sure that's the intent :)

Reply:
Just refactored this because I think it's better to just return the default version in those cases.